packages/common/src/dynamicLoading/dynamic-loading-setting.ts

```diff
@@ -0,0 +1,73 @@
+import type { FontSource } from './types';
+
+declare global {
```

Comment:
Is this to avoid adding Expo as a dev dependency? If so, can we add a comment about where it comes from so it's easy to check if it's still accurate?

Reply:
This is to make TS understand the signatures of the functions. I will add comments for where those come from.